package.json

```diff
@@ -9,7 +9,7 @@
     "dist/"
   ],
   "scripts": {
-    "prepare": "yarn build",
+    "prepack": "[[ ! -d ./dist ]] && echo \"Cannot create package: missing 'dist' folder\" && exit 1",
```

Comment:
where this "prepack" script will be called?

Reply:
Before `npm pack` is executed, and that includes `npm publish`. See https://docs.npmjs.com/cli/v6/using-npm/scripts#prepare-and-prepublish

Reply:
thanks!